templates/pgbouncer/pgbouncer-networkpolicy.yaml

```diff
@@ -40,6 +40,16 @@ spec:
 {{- end }}
         matchLabels:
           app: keda-operator
+{{- if .Values.workers.keda.namespaceLabels }}
```

Comment:
Line 33-42 is almost exactly a duplicate of this. Meaning if `.Values.workers.keda.namespaceLabels` is set we'd end up with them twice in the output. That doesn't seem right.

Reply:
It should be the same because the operator and the apiserver should never live in different namespaces. Up until this version these were actually the same pod, so they're very tightly coupled.

Reply:
Ohh, this is two separate selectors. Gotcha.